deploy/common/k8sbased.sh

```diff
@@ -680,36 +700,39 @@ function launch_sensor {
         rm "$k8s_dir/sensor-deploy.zip"
       fi
 
-      namespace=stackrox
-      if [[ -n "$NAMESPACE_OVERRIDE" ]]; then
-        namespace="$NAMESPACE_OVERRIDE"
-        echo "Changing namespace to $NAMESPACE_OVERRIDE"
-        ls $k8s_dir/sensor-deploy/*.yaml | while read file; do sed -i'.original' -e 's/namespace: stackrox/namespace: '"$NAMESPACE_OVERRIDE"'/g' $file; done
-        sed -itmp.bak 's/set -e//g' $k8s_dir/sensor-deploy/sensor.sh
+      if [[ -n "${NAMESPACE_OVERRIDE}" ]]; then
+        if [[ "${namespace}" != "stackrox" && "${namespace}" != "${NAMESPACE_OVERRIDE}" ]]; then
```

Comment:
Being careful not to break existing workflows while still supporting the new way of setting namespaces. Actually this makes `NAMESPACE_OVERRIDE` superfluous.